javascript/javascript3/week1/gif.js

```diff
@@ -0,0 +1,13 @@
+const searchGif = () => {
+    const searchWord = document.getElementById("search-input").value;
+    fetch(`https://api.giphy.com/v1/gifs/search?api_key=SMnr7cJBz7A2NPHAeprGNJ0oHb0796pC&q=${searchWord}`)
+      .then(response => response.json())
+      .then(data => {
+        const gifUrl = data.data[0].images.original.url;
+        const gifImg = document.createElement("img");
+        gifImg.src = gifUrl;
+        document.getElementById("gif-container").appendChild(gifImg);
+      })
+      .catch(error => console.log(error));
+  };
+  
```

Comment:
One more thing. 
Add an input element, where the user can specify how many gif results the user wants. That what was in the task, probably you forgot about it 😊